bin/init-s3.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+awslocal s3 mb s3://willa
```

Comment:
somehow, this should inherit the value for `LANCEDB_URI` for the app container if it's got `s3://`, and if not, default to `s3://willa`.